packages/react-reconciler/src/__tests__/__snapshots__/ReactIncrementalPerf-test.js.snap

```diff
@@ -1,7 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ReactDebugFiberPerf captures all lifecycles 1`] = `
-"// Mount
+"⚛ (Waiting for async callback...)
+  // Mount
```

Comment:
These comments are a bit confusing now. They were meant to be attached to the phases below. Any way we can avoid nesting them?